srcpkgs/python3-dulwich/template

```diff
@@ -1,26 +1,24 @@
 # Template file for 'python3-dulwich'
 pkgname=python3-dulwich
```

Comment:
If this were in a separate PR, which it probably should have been by the contributing guidelines, I'd merge this package already.

Reply:
Good point!